requirements/testing.txt

```diff
@@ -17,7 +17,7 @@ execnet==1.8.0
 # supported platform.
 # mercurial-scm.org/wiki/SupportedPythonVersions
 # (Pinned to 4.4.2 since what we need for testing is still useful)
-Mercurial==4.4.2  # pyup: ignore
+# Mercurial==4.4.2  # pyup: ignore
```

Comment:
For some reason, this kept failing, but everything else worked when I commented it out. I'm sure this is used elsewhere so I know this should be handled a better way